vendor/bat-native-confirmations/src/bat/confirmations/internal/ad_grants.cc

```diff
@@ -45,12 +45,13 @@ bool AdGrants::SetFromJson(const std::string& json) {
 bool AdGrants::SetFromDictionary(base::DictionaryValue* dictionary) {
   DCHECK(dictionary);
 
-  auto* value = dictionary->FindKey("grants_balance");
-  if (!value || !value->is_double()) {
+  const base::Optional<double> balance =
```

Comment:
Unit tests highlighted that `0` in the `confirmations.json` state file for the `balance` would return `false` due to `is_double()` failing